### PR TITLE
updated hindi readme.md

### PR DESCRIPTION
**Pull request by DhruvMishra1826**
@akkshayTandon i have updated some links and images which were not working earlier in hindi Readme.md. Please review it.

### CODE REVIEW

**Comment by CandidDeer** [comment]
@Syknapse Please review this general enhancement and let me know whether I should merge. Thx.

**Comment by DhruvMishra1826** [comment]
> * on line no. 101,  the image is not showing
> 
>     * on line no. 287, the link is broken

Ok, i will update that.

File: translations/README/HINDI.md [hunk 1]
```diff
@@ -98,13 +98,13 @@ _अनुमानित समय: 30 मिनट से कम_.
 - अब हम परियोजना की एक स्थानीय प्रति बनाना चाहते हैं। यह आपकी अपनी मशीन पर सहेजी गई एक प्रति है।
 - GitHub डेस्कटॉप ऐप खोलें। ऐप में:
 
-| <ul><li>_File_ पर क्लिक करें, फिर _Clone रिपॉजिटरी_</li></ul> | पर क्लिक करें ![क्लोन](readme-only/clone.पीएनजी 'क्लिक क्लोन रिपॉजिटरी') |
+| <ul><li>_File_ पर क्लिक करें, फिर _Clone रिपॉजिटरी_</li></ul> | पर क्लिक करें ![क्लोन](/readme-only/clone.पीएनजी 'क्लिक क्लोन रिपॉजिटरी') |
```
Comment (TenTraicion) [inline translations/README/HINDI.md:101]:
```suggestion
| <ul><li>_File_ पर क्लिक करें, फिर _Clone रिपॉजिटरी_</li></ul> | पर क्लिक करें ![क्लोन](/readme-only/clone "पीएनजी 'क्लिक क्लोन रिपॉजिटरी'") |
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@DhruvMishra1826, maybe this will do the trick, also about Line 287, removing space in the link will fix it. `https:/  /`

**Comment by Syknapse** [comment]
> @Syknapse Please review this general enhancement and let me know whether I should merge. Thx.

Once the changes requested are done, we can go ahead and merge, thank you

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@DhruvMishra1826, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**